charts/newrelic-logging/templates/daemonset.yaml

```diff
@@ -28,9 +28,9 @@ spec:
         release: {{.Release.Name }}
         kubernetes.io/os: linux
         app.kubernetes.io/name: {{ template "newrelic-logging.name" . }}
-        {{- if .Values.podLabels}}
-        {{ toYaml .Values.podLabels }}
-        {{- end }}
+{{- if .Values.podLabels}}
```

Comment:
I think that following other if + indent blocks on this file, we should keep the if and end at the right indentation and just move the toYaml to the beginning of the line